src/components/loading.tsx

```diff
@@ -0,0 +1,22 @@
+import { AnimatedLoader } from '@powerhousedao/design-system';
+import { useLoadingScreen } from 'src/hooks/useLoadingScreen';
+import { twMerge } from 'tailwind-merge';
+
```

Comment:
In a case like this, we should try favor putting this component in the design system. the `showLoadingScreen` and `loadingComponent` can be passed as props. Otherwise we will always have to maintain our UI code in multiple places. For example, if we changed the color indicated by `bg-white` used here in the `design-system`, then we would also have to update it here. Not saying anything needs changing now, but just to think about for the future 👍